### PR TITLE
Fixes Glass booze bottles and molotovs not splashing their contents when broken via throwing

### PR DESCRIPTION
**Pull request by tohg7**
Since most other codebases' booze bottles splash their contents when broken via throwing them at things, i thought i'd straighten this inconsistency out so that booze bottles can splash when smashed via throwing them at things since i noticed that  this doesn't happen in-game for this codebase although drinking glasses splash when smashed via being thrown at things. Quite the inconsistency, i decided to fix it.
## About The Pull Request
Makes all booze bottles and Molotov cocktails splash their contents upon being smashed via throwing them at things, fixing an inconsistency where shot/drinking glasses would splash upon being smashed via throw while booze bottles and molotovs would not.
Footage is right here: https://streamable.com/rjg7dr

## Why It's Good For The Game
Molotov Cocktails currently are currently seen as "not worth using" by most of the playerbase due to only being able to light one tile on fire for a brief moment on impact and not splashing their contents when thrown, by fixing this inconsistency between shot/drinking glasses and booze bottles/molotovs, this makes them slightly more powerful. 

## Changelog

:cl:
fix: fixed an inconsistency where booze bottles and molotovs wouldn't splash their contents when thrown at things while drinking and shot glasses do.
/:cl:

### CODE REVIEW

**Comment by BallastMonster** [comment]
congrats on ya first PR

**Comment by Fikou** [comment]
the changelog things are supposed to be lowercase

File: code/modules/food_and_drinks/drinks/drinks/bottle.dm [hunk 1]
```diff
@@ -20,6 +20,7 @@
 	righthand_file = 'icons/mob/inhands/misc/food_righthand.dmi'
 	isGlass = TRUE
 	foodtype = ALCOHOL
+	resistance_flags = ACID_PROOF
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/food_and_drinks/drinks/drinks/bottle.dm:23]: not documented

File: code/modules/food_and_drinks/drinks/drinks/bottle.dm [hunk 2]
```diff
@@ -140,6 +141,7 @@
 	throw_speed = 3
 	throw_range = 5
 	w_class = WEIGHT_CLASS_TINY
+	custom_materials = list(/datum/material/glass=350)
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/food_and_drinks/drinks/drinks/bottle.dm:144]: not documented

**Review by ExcessiveUseOfCobblestone — COMMENTED** [review]
were these changes you wanted to make here?

**Comment by Wallemations** [comment]
> were these changes you wanted to make here?

they put it in their changelog but didn't say they did it in the PR itself

**Comment by tohg7** [comment]
> > were these changes you wanted to make here?
> 
> they put it in their changelog but didn't say they did it in the PR itself

yes, those were part of the changes that i wanted to make.

**Comment by Mothblocks** [comment]
image

Did you really submit a fix PR so you could have what in your eyes seems to be a weapon with "no weakness"?

**Comment by LemonInTheDark** [comment]
Absolutely fucking cringe

**Comment by LemonInTheDark** [comment]
> image
> 
> Did you really submit a fix PR so you could have what in your eyes seems to be a weapon with "no weakness"?

~~He put it in the comment right above you~~

**Comment by tohg7** [comment]
> image
> 
> Did you really submit a fix PR so you could have what in your eyes seems to be a weapon with "no weakness"?

upon further examination, it doesn't really have "no weaknesses" but rather a few weaknesses, if you wear a regular gas mask while wearing a helmet, you can block the capsaicin and the bottle smash.

**Comment by LEtranger135** [comment]
Hello, i'm the coder who made this PR, if you're wondering if it's really me, i merely forgot my password after my data was cleared to free up memory.

**Pull request merged**